app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
```

Comment:
Common practice for empty methods in rails is: 
`def new; end`

just saves some extra lines